learn-more.html

```diff
@@ -2,12 +2,163 @@
 <html>
   <head>
     <title>Learn More</title>
+    <style>
+      @import "index.css";
+    </style>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
   </head>
   <body>
-    <section id="header"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
+    <section id="header2">

```

Comment:
rather then doing things like "header2" or "main2" we could do this as "learnMoreHeader" or "learnMoreMain" (referencing line 17 as well)